catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -208,4 +225,48 @@ public void setFeatureService(FeatureService featureService) {
   public void setEndpointUtil(EndpointUtil util) {
     this.util = util;
   }
+
+  private Map<String, Object> getLatLonLiteral(String query) {
+    String queryStringNumbersOnly = query.replaceAll("[^\\.\\-\\d]", " ");
```

Comment:
Intellij says that your regex is a little redundant and can be replaced by `[^.\\-\\d]`.